core/micro-services/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```

Comment:
should this be gitignored?

Reply:
no we should commit it to fix sbt version